include/Epoll.hpp

```diff
@@ -0,0 +1,30 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   Epoll.hpp                                          :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: bthomas <bthomas@student.42.fr>            +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/09/24 12:21:12 by bthomas           #+#    #+#             */
+/*   Updated: 2024/09/24 13:14:21 by bthomas          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#pragma once
+#ifndef EPOLL_HPP
+# define EPOLL_HPP
+
+#include "Socket.hpp"
+
+class Epoll
+{
+	private:
+		int _epollFd;
+		struct epoll_event _event;
```

Comment:
there could be a lot of `epoll_event`, did you add it as a member to re-use the same variable every time?

Reply:
good point, might need to change this